src/app/(dev)/account/page.tsx

```diff
@@ -1,3 +1,30 @@
-export default function AccountPage() {
-  return <div>Account Page</div>;
+import React from "react";
+
+import type { CurrentUserProps } from "@/lib/hoc/withCurrentUser";
+import withCurrentUser from "@/lib/hoc/withCurrentUser";
+import { getUserFromId } from "@/services/user";
+import type { PlayLevel } from "@/types/types";
+import ClientAccountPage from "./client-page";
+
+export const metadata = {
+  title: "Account Settings - UABC Booking Portal",
+};
+
+async function AccountPage({ currentUser }: CurrentUserProps) {
+  const user = await getUserFromId(currentUser.id);
```

Comment:
to prevent optional chaining, you could enforce that the user, and required fields all exist, and `redirect("auth/login")` if not